### ISSUE TITLE
arduino-pico: Full RP2040 Web Support

### ISSUE THREAD

**Issue by sgbaird** [MrYsLab/Telemetrix4RpiPicoW#1]
https://github.com/earlephilhower/arduino-pico/releases/tag/2.5.0

Not sure if this is related

**Comment by MrYsLab** [comment]
I am not sure how I would use this. What are your thoughts?

**Comment by sgbaird** [comment]
Ah gotcha. Nvm, I've just been looking into transferring data between RPi Pico W and another device via a web server

**Comment by MrYsLab** [comment]
It is relatively easy to transfer HTTP data using Python's [HTTP support](https://docs.python.org/3/library/http.html). Python also provides urllib and requests. 

This suggestion may be overkill, but I have a framework that allows multiple computers to communicate with one another. It is called [python-banyan](https://mryslab.github.io/python_banyan/users_guide/).

I used a precursor to this framework to control a Sparkfun RebBot robot with this [webpage](http://mryslab.github.io/rbDashBoard/).

I created a web page to control several different hardware types using python-banyan. To see what the Arduino control program looked like, search for "Banyan Demonstration Station Web Pages" on [this page](https://mryslab.github.io/python_banyan/one_gpio_demos/#running-the-web-demo-station-for-the-arduino), and you will see some screenshots.
